app/controllers/admin/cycles_controller.rb

```diff
@@ -72,9 +72,22 @@ def new
 
   def create
```

Comment:
Yeah, since this PR is a simple correction, I will add another issue so we can work on this later